File: All-Topics/751-800/771-JewelsandStones.ts
const jewels = "aA",
  stones = "aAAbbbb"

/**
 * @param {string} jewels
 * @param {string} stones
 * @return {number}
 */
var numJewelsInStones = function (jewels: string, stones: string) {
  let result = 0
  const stonesLength = stones.length

  for (let i = 0; i < stonesLength; i++) {
    if (jewels.includes(stones[i])) result++
  }
  return result
}

console.log(numJewelsInStones(jewels, stones))

File: All-Topics/751-800/771-JewelsandStones.java
class Solution {
  public int numJewelsInStones(String jewels, String stones) {
    String newStones = stones;
    for (int i = 0; i < jewels.length(); i++) {
      newStones = newStones.replace(String.valueOf(jewels.charAt(i)), "");
    }

    return stones.length() - newStones.length();
  }
}